server/router/cohort-router.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../controllers/cohort/getCohorts');
```

Comment:
@alidah0 This file might need to export named routes in the future instead of a default export, to be useful.  You could do that now, or you could do that later 🤷‍♂️